src/CTA.FeatureDetection.Common/WCFConfigUtils/Constants.cs

```diff
@@ -0,0 +1,51 @@
+﻿
+namespace CTA.FeatureDetection.Common.WCFConfigUtils
+{
+    public class Constants
+    {
+        internal const string SystemServiceModelElement = "system.serviceModel";
+        internal const string WCFClientElement = "client";
+        internal const string WCFServiceElement = "services";
+        internal const string WCFServiceEndpoint = "endpoint";
+        internal const string BindingElement = "element";
+        internal const string ServiceContractAttribute = "ServiceContractAttribute";
+        internal const string OperationContractAttribute = "OperationContractAttribute";
+        internal const string BindingsAttribute = "bindings";
+        internal const string BindingAttribute = "binding";
+        internal const string SecurityElement = "security";
+        internal const string ModeAttribute = "mode";
+        internal const string ProtocolMappingAttribute = "protocolMapping";
+        internal const string AddElement = "add";
+
+        internal const string AddServiceEndpointType = "AddServiceEndpoint";
+        internal const string EndpointAddressType = "EndpointAddress";
+        internal const string SystemServiceModelClass = "System.ServiceModel";
+        internal const string ServiceEndpointClass = "ServiceEndpoint";
+
+        internal const string NetTcpBinding = "nettcpbinding";
```

Comment:
convert to enum

Reply:
Not used enums here because while storing binding, the Dictionary would ideally become Dictionary<BindingType, BindingConfiguration> , but it might not necessarily be one of the 5 we support. Let me know your thoughts on this.